src/User.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const User = ({ user }) => (
+  <div>
```

Comment:
use React fragment if you don't really need wrapper tags

Reply:
when i use fragment it ruines my pretty styles)